composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "keboola/db-extractor-ssh-tunnel",
+    "description": "Create SSH tunnel for DB Extractors",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "keboola/ssh-tunnel": "^1.0|^2.0",
+        "keboola/db-extractor-logger": "dev-ondra-init-repo",
+        "keboola/retry": "^0.3"
+    },
+    "require-dev": {
+        "keboola/coding-standard": "^2.0",
```

Comment:
Proč? 

https://packagist.org/packages/keboola/coding-standard#8.0.0, dtto phpstan (a proč ne phpstan-shim) a phpunit

Reply:
Normální jsou jasné - tam je to kvuli kompatibilitě s common a starými extraktory. Ale pro require-dev je to úplně jedno.

Reply:
nastavil jsem tam phpstan-shim místo phpstan a nejnovější knihovny phpunit a keboola/coding-standard